install

```diff
@@ -384,7 +384,6 @@ Dir.chdir HOMEBREW_REPOSITORY do
     curl_flags += "k" if mac? && macos_version < "10.6"
     core_tap = "#{HOMEBREW_PREFIX}/Homebrew/Library/Taps/homebrew/homebrew-core"
     system "/bin/bash -o pipefail -c 'curl -#{curl_flags} #{BREW_REPO}/tarball/master | /bin/tar xz -m --strip 1'"
-
```

Comment:
Please don't delete the blank line here. The empty blank line is desirable. Previously you also had spaces on this line.

Reply:
@sjackman Oops! 
Why? As far as I know there is no deference since the command will skip that line! 
But I think it fixed now!